src/main.rs

```diff
@@ -49,6 +55,24 @@ fn print_total(total: usize) {
         problems += "s";
     }
 
+    println!(
+        "\n{}",
+        format!(
+            "{} {} {}",
+            String::from("Found").red().bold(),
+            total.to_string().red().bold(),
+            problems.red().bold()
+        )
```

Comment:
Can we rewrite like that?
```suggestion
        format!(
            "{} {} {}",
            String::from("Found"),
            total.to_string(),
            problems
        ).red().bold()
```